src/com/nutrons/framework/Robot.java

```diff
@@ -81,6 +87,10 @@ public final void startCompetition() {
   protected final void disabled() {
   }
 
+  public Command registerAuto() {
```

Comment:
@totallyashes The CI failed to build, but when I reran it, things built without incident.  Is this potentially concurrency related?